workspaces/adventure-pack/goodies/python3/src/flatten/test.py

```diff
@@ -0,0 +1,19 @@
+from . import *
+
+
+def test_flatten():
+    l = [[0, 1, 2, 3], [[4, 5, 6], [7, 8, 9]], 10, 11, 12]
```

Comment:
One principle I recommend for test data is to try to avoid obvious patterns. For example, in this case, because all the numbers are consecutive, if the implementation accidentally yielded indexes instead of values, the test would still pass.

Of course you have other tests that would fail, but when it's easy to do so I recommend making the test data more naturally strong. (If you need inspiration, I am a big fan of pi digits, or any other numbers you enjoy work too.)

A fun way to think about writing tests imo is to imagine the implementer as an adversary trying to outwit you by writing a wrong implementation that still passes the tests. As the test writers our job is to make that difficult 😄